en/advanced/computer_vision.md

```diff
@@ -0,0 +1,49 @@
+# Computer Vision (VIO, Avoidance)
+
+[Computer vision](https://en.wikipedia.org/wiki/Computer_vision) techniques enable computers to use visual data to make sense of their environment. 
+
+PX4 uses computer vision systems (primarily running on [Companion Computers](../companion_computer/pixhawk_companion.md)) in order to support the following features:
+- [Optical Flow](#optical_flow) provides 2D velocity estimation (using a downward facing camera and a downward facing distance sensor).
+- [Motion Capture](#mocap) provides 3D pose estimation using a vision system that is *external* to the vehicle. 
+  It is primarily used for indoor navigation.
+- [Visual Inertial Odometry](#vio) provides 3D pose and velocity estimation using an onboard vision system and IMU. 
+  It is used for navigation when global position information is absent or unreliable.
+- [Obstacle Avoidance](https://docs.px4.io/en/computer_vision/obstacle_avoidance.html) provides navigation around obstacles when flying a planned path (currently missions are supported). This uses [PX4/avoidance](https://github.com/PX4/avoidance) running on a companion computer.
+- [Collision Prevention](https://docs.px4.io/en/computer_vision/collision_prevention.html) is used to stop vehicles before they can crash into an obstacle (primarily when flying in manual modes).
+
+
+## Motion Capture {#mocap}
+
+Motion Capture (MoCap) is a technique for estimating the 3D *pose* (position and orientation) of a vehicle using a positioning mechanism that is *external* to the vehicle.
+MoCap systems most commonly detect motion using infrared cameras, but other types of cameras, Lidar, or Ultra Wideband (UWB)  may also be used. 
+
+> **Note** MoCap is commonly used to navigate a vehicle in situations where GPS is absent (e.g. indoors), and provides position relative to a a *local* co-ordinate system.
+
+For information about MoCap see:
+- [External Position Estimation](../ros/external_position_estimation.md)
+- [Flying with Motion Capture (VICON, Optitrack)](../tutorials/motion-capture-vicon-optitrack.md)
+- [EKF > External Vision System](../tutorials/tuning_the_ecl_ekf.md#external-vision-system)
+
+
+## Visual Inertial Odometry {#vio}
+
+Visual Inertial Odometry (VIO) is used for estimating the 3D *pose* (position and orientation) of a moving vehicle relative to a *local* starting position.
+It is commonly used to navigate a vehicle in situations where GPS is absent (e.g. indoors) or unreliable (e.g. when flying under a bridge).
+
+VIO uses [Visual Odometry](https://en.wikipedia.org/wiki/Visual_odometry) to estimate vehicle *pose* from visual information, combined with inertial measurements from an IMU (to correct for errors associated with rapid vehicle movement resulting in poor image capture).
+
+> **Note** VIO uses some similar techniques to [MoCap](#mocap). One difference is that the VIO cameras/IMU are vehicle-based, and additionally provide velocity information.
```

Comment:
Can you clarify this? I don't see how the techniques are similar.

Reply:
That was my interpretation of your comments - ie that they both use image analysis to get position/pose. Because I don't understand the detail I used the "weasel words" *some similar* :-). 
I've updated this so that it just refers to the on/off vehicle camera difference.